### DISCUSSION TITLE
Is there a reference for the default layout?

### DISCUSSION THREAD

**Discussion by mankoff** [manna-harbour/miryoku#154] [Q&A] [answered]
I've been looking at the default layout here:
![image](https://user-images.githubusercontent.com/145117/195003887-b6e8a42c-eb9a-4e23-8be2-57664439e428.png)

But I'm unable to figure out certain keys. Is there a reference?

For example, I don't know what `Tap`, `Extra`, or `Base` mean, nor how to access / activate them. What about `Opp` and `Cur`.

It looks like if I use left thumb Media that the right side has volume up/down. I can verify this on my laptop, it displays the volume change overlay. But I also see (bottom left of keys) the symbols: 3, 2, 1, 0, O, E, R, M, H, S, V. Why are these on the media layer? What do they do? I don't see any actions on my computer when I press them.

**Comment by manna-harbour** [comment, marked as answer]
That diagram is just a combined layer reference for use when you already understand the layers. The individual layers are described at  https://github.com/manna-harbour/miryoku/tree/master/docs/reference#layers.
